source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/ActionSetRendererTests.swift

```diff
@@ -77,6 +77,24 @@ class ActionSetRendererTests: XCTestCase {
         XCTAssertEqual(actionSetView.alignment, .centerY)
     }
     
+    func testNumberOfAction() {
+        let hostConfigActions = ACSActionsConfig(showCard: .init(actionMode: .inline, style: .accent, inlineTopMargin: 0), actionsOrientation: .vertical, actionAlignment: .center, buttonSpacing: 2, maxActions: 5, spacing: .default, iconPlacement: .aboveTitle, iconSize: .init(value: 1))
+        hostConfig = .make(actions: hostConfigActions)
```

Comment:
```suggestion
        let hostConfigActions = ACSActionsConfig(showCard: FakeShowCard(actionMode: .inline, style: .accent, inlineTopMargin: 0), actionsOrientation: .vertical, actionAlignment: .center, buttonSpacing: 2, maxActions: 5, spacing: .default, iconPlacement: .aboveTitle, iconSize: .init(value: 1))
        hostConfig = .make(actions: hostConfigActions)
```

Use class names instead of .init